angular/test/test-app/src/app/overlay-auto-mount/overlay-auto-mount.component.ts

```diff
@@ -0,0 +1,13 @@
+import { Component } from "@angular/core";
+
+/**
+ * Validates that inline modals correctly mount
+ * inner components when keepContentsMounted is
+ * enabled.
+ */
+@Component({
+  selector: 'app-overlay-auto-mount',
```

Comment:
~ File name, selector and class name should have the same naming conventions. 

I would recommend the class be called: `OverlayKeepContentsMountedComponent` to align with Angular style guides for component naming.